### ISSUE TITLE
Unable to build docker image

### ISSUE THREAD

**Issue by k3z** [mvt-project/mvt#536]
Hi,

After attempting to install `mvt` natively on macOS Sequoia, which did not work, I was unable to install `libimobiledevice`.
I turned to the Docker version and encountered this error (full trace below) during the build:

No package 'libtatsu-1.0' found

The required dependencies for iOS seem to cause issues in all versions.

Is there an alternative method on macOS?

Thank you very much for your help and your work.

```
25.63 checking for libtatsu-1.0 >= 1.0.3... no
25.63 configure: error: Package requirements (libtatsu-1.0 >= 1.0.3) were not met:
25.63
25.63 No package 'libtatsu-1.0' found
25.63
25.63 Consider adjusting the PKG_CONFIG_PATH environment variable if you
25.63 installed software in a non-standard prefix.
25.63
25.63 Alternatively, you may set the environment variables libtatsu_CFLAGS
25.63 and libtatsu_LIBS to avoid the need to call pkg-config.
25.63 See the pkg-config man page for more details.
------
Dockerfile:39
--------------------
  38 |     # ----------------------
  39 | >>> RUN git clone https://github.com/libimobiledevice/libplist \
  40 | >>>   && git clone https://github.com/libimobiledevice/libimobiledevice-glue \
  41 | >>>   && git clone https://github.com/libimobiledevice/libusbmuxd \
  42 | >>>   && git clone https://github.com/libimobiledevice/libimobiledevice \
  43 | >>>   && git clone https://github.com/libimobiledevice/usbmuxd \
  44 | >>>
  45 | >>>   && cd libplist && ./autogen.sh && make && make install && ldconfig \
  46 | >>>
  47 | >>>   && cd ../libimobiledevice-glue && PKG_CONFIG_PATH=/usr/local/lib/pkgconfig ./autogen.sh --prefix=/usr && make && make install && ldconfig \
  48 | >>>
  49 | >>>   && cd ../libusbmuxd && PKG_CONFIG_PATH=/usr/local/lib/pkgconfig ./autogen.sh && make && make install && ldconfig \
  50 | >>>
  51 | >>>   && cd ../libimobiledevice && PKG_CONFIG_PATH=/usr/local/lib/pkgconfig ./autogen.sh --enable-debug && make && make install && ldconfig \
  52 | >>>
  53 | >>>   && cd ../usbmuxd && PKG_CONFIG_PATH=/usr/local/lib/pkgconfig ./autogen.sh --prefix=/usr --sysconfdir=/etc --localstatedir=/var --runstatedir=/run && make && make install \
  54 | >>>
  55 | >>>   # Clean up.
  56 | >>>   && cd .. && rm -rf libplist libimobiledevice-glue libusbmuxd libimobiledevice usbmuxd
  57 |
--------------------
ERROR: failed to solve: process "/bin/sh -c git clone https://github.com/libimobiledevice/libplist   && git clone https://github.com/libimobiledevice/libimobiledevice-glue   && git clone https://github.com/libimobiledevice/libusbmuxd   && git clone https://github.com/libimobiledevice/libimobiledevice   && git clone https://github.com/libimobiledevice/usbmuxd   && cd libplist && ./autogen.sh && make && make install && ldconfig   && cd ../libimobiledevice-glue && PKG_CONFIG_PATH=/usr/local/lib/pkgconfig ./autogen.sh --prefix=/usr && make && make install && ldconfig   && cd ../libusbmuxd && PKG_CONFIG_PATH=/usr/local/lib/pkgconfig ./autogen.sh && make && make install && ldconfig   && cd ../libimobiledevice && PKG_CONFIG_PATH=/usr/local/lib/pkgconfig ./autogen.sh --enable-debug && make && make install && ldconfig   && cd ../usbmuxd && PKG_CONFIG_PATH=/usr/local/lib/pkgconfig ./autogen.sh --prefix=/usr --sysconfdir=/etc --localstatedir=/var --runstatedir=/run && make && make install   && cd .. && rm -rf libplist libimobiledevice-glue libusbmuxd libimobiledevice usbmuxd" did not complete successfully: exit code: 1

 3 warnings found (use docker --debug to expand):
 - JSONArgsRecommended: JSON arguments recommended for CMD to prevent unintended behavior related to OS signals (line 80)
 - NoEmptyContinuation: Empty continuation line (line 34)
 - NoEmptyContinuation: Empty continuation line (line 56)

View build details: docker-desktop://dashboard/build/desktop-linux/desktop-linux/lzbwyvophszh12avioxd9d86f
```

**Comment by scribblemaniac** [comment]
Yes, this has been broken for a few months. The dockerfiles in my PR do not have this issue if you would like to test them: #509

**Comment by DonnchaC** [comment]
I've merged #509 which should fixed it for you. Can you try it and reopen if this is still a problem?

**Comment by k3z** [comment]
Thank you for your help. 
I manage to build docker image with current main branch (v2.5.4)
